airflow/providers/amazon/aws/executors/batch/__init__.py

```diff
@@ -14,3 +14,8 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
+from __future__ import annotations
+
+from airflow.providers.amazon.aws.executors.batch.batch_executor import AwsBatchExecutor
+
+AwsBatchExecutor = AwsBatchExecutor
```

Comment:
Precommit static tests also required that the import be used and suggested using a "redundant alias", so I went with that approach rather than adding a #no-qa type ignore.

Reply:
Very interesting, thanks would have been scratching my head on that one without this comment.

Reply:
I wonder if it's worth a comment?

Reply:
Sure, yeah, that's fair. I'll some comments to the code itself.